spacy/pipeline/entity_linker.py

```diff
@@ -220,16 +226,15 @@ def __init__(
         self.model = model
         self.name = name
         self.labels_discard = list(labels_discard)
+        # how many neighbour sentences to take into account
         self.n_sents = n_sents
         self.incl_prior = incl_prior
         self.incl_context = incl_context
         self.get_candidates = get_candidates
         self.get_candidates_batch = get_candidates_batch
         self.cfg: Dict[str, Any] = {"overwrite": overwrite}
         self.distance = CosineDistance(normalize=False)
-        # how many neighbour sentences to take into account
-        # create an empty KB by default
-        self.kb = empty_kb(entity_vector_length)(self.vocab)
+        self.kb = generate_empty_kb(self.vocab, entity_vector_length)
```

Comment:
`empty_kb()` couldn't be configured. The default implementation for `generate_empty_kb()` mirrors the behavior of the default implementation `empty_kb()`, the only difference being a slightly different signature. So this should be backwards-compatible IMO.